docs/common/message-workflows.md

```diff
@@ -0,0 +1,314 @@
+# Message Workflows
```

Comment:
please also add "this page was migrated" here

Reply:
Resolved in 0268958.